AuthenticationService/src/main/java/org/energygrid/east/authenticationservice/model/enums/AccountRole.java

```diff
@@ -0,0 +1,9 @@
+package org.energygrid.east.authenticationservice.model.enums;
+
+public enum AccountRole {
+    Customer,
+    LargeScaleCustomer,
+    UtilityCompany,
+    ResponsibleParty,
+    Admin
```

Comment:
```suggestion
    CUSTOMER,
    LARGE_SCALE_CUSTOMER,
    UTILITY_COMPANY,
    RESPONSIBLE_PARTY,
    ADMIN
```
Enums in Java are always capitalised